Misc/NEWS.d/next/Core and Builtins/2020-01-30-14-36-31.bpo-39502.IJu0rl.rst

```diff
@@ -0,0 +1,2 @@
+Fix PyTime_localtime() on AIX so that only 32-bit is range tested for overflow.
```

Comment:
I prefer to mention time.localtime() which is seen by users, than mention a private C function. 32-bit overflow is also obscure. I propose:

```
Fix :func:`time.localtime` on 64-bit AIX  to support years before 1902 and after 2038.
Patch by M Felt.
```